cognite/cli/cli.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import argparse
+import sys
+
+from cognite.cli.cli_models import CogniteMLCLI
+
+
+class CogniteCLI:
+    def __init__(self):
+        """cognite <service> [<args>]
+
+The available Cognite services are:
+    models      Model hosting
+"""
+        parser = argparse.ArgumentParser(description="Command line interface for CDP.", usage=self.__init__.__doc__)
+        parser.add_argument("service", help="Service to run", choices=["models"])
```

Comment:
Maybe you can use sub-commands? https://docs.python.org/3/library/argparse.html#sub-commands